util/dvsim/SimCfg.py

```diff
@@ -659,6 +660,14 @@ def create_bucket_report(buckets):
                     results_str += self.cov_report_deploy.cov_results
                     self.results_summary[
                         "Coverage"] = self.cov_report_deploy.cov_total
+
+                    # Copy cov_report to the report directory and create a
+                    # symlink to the latest directory.
+                    dest_target_dir = os.path.join(self.results_path, "cov_report")
+                    dest_symlink_dir = os.path.join(self.symlink_path, "cov_report")
+                    mk_path(dest_symlink_dir)
+                    shutil.copytree(self.cov_report_dir, dest_target_dir)
```

Comment:
IMO this is better handled in `CovReport::post_finish()` in `Deploy.py`. It already maintains `cov_report_dir` as instance variable.